otelcollector/deploy/addon-chart/ccp-metrics-plugin/values-template.yaml

```diff
@@ -0,0 +1,122 @@
+AzureMonitorMetrics:
+  KubeStateMetrics:
+# Uncomment the below 2 settings if you want to pass in the labels and/or annotations
+    #MetricLabelsAllowlist: "testlabel=[.*]"
+    #MetricAnnotationsAllowList: ""
+    ImageRepository: "/oss/kubernetes/kube-state-metrics"
+# Kube-state-metrics ImageTag - 2.9.2, corresponds to chart version - 5.10.1
+    ImageTag: "v2.9.2"
+    Collectors:
+      - certificatesigningrequests
+      - configmaps
+      - cronjobs
+      - daemonsets
+      - deployments
+      - endpoints
+      - horizontalpodautoscalers
+      - ingresses
+      - jobs
+      - leases
+      - limitranges
+      - mutatingwebhookconfigurations
+      - namespaces
+      - networkpolicies
+      - nodes
+      - persistentvolumeclaims
+      - persistentvolumes
+      - poddisruptionbudgets
+      - pods
+      - replicasets
+      - replicationcontrollers
+      - resourcequotas
+      - secrets
+      - services
+      - statefulsets
+      - storageclasses
+      - validatingwebhookconfigurations
+      - volumeattachments
+  ImageRepository: mcr.microsoft.com${MCR_REPOSITORY}
+  ImageTag: ${IMAGE_TAG}
+  ImageTagWin: ${IMAGE_TAG}-win
+  # The below 2 settings are not Azure Monitor Metrics adapter chart. They are substituted in a different manner.
+  # Please update these with the latest ones from here so that you get the image that is currently deployed by the AKS RP -
+  # Repository: https://msazure.visualstudio.com/CloudNativeCompute/_git/aks-rp?path=/ccp/charts/addon-charts/azure-monitor-metrics-addon/templates/ama-metrics-daemonset.yaml&version=GBrashmi/prom-addon-arm64&line=136&lineEnd=136&lineStartColumn=56&lineEndColumn=85&lineStyle=plain&_a=contents
+  # ImageTag: https://msazure.visualstudio.com/CloudNativeCompute/_git/aks-rp?path=/ccp/charts/kube-control-plane/templates/_images.tpl&version=GBrashmi/prom-addon-arm64&line=530&lineEnd=530&lineStartColumn=28&lineEndColumn=53&lineStyle=plain&_a=contents
+  AddonTokenAdapter:
```

Comment:
we could remove a bunch of things from this file but maybe is better to keep it the same as the addon?

Reply:
up to you, it will be moving to aks-rp soon, can tidy it here now or during the move. Perhaps if there are things we know we don't need (like ksm values) remove them now, but be conservative with what is removed.

Reply:
I'll remove them now that I've been re-building images for testing